test/sr_test/sr_sessions.erl

```diff
@@ -130,6 +131,9 @@ new(User) ->
    , expires_at => expires_at()
    }.
 
+-spec id(session()) -> id().
```

Comment:
This module was intended **not** to implement an `id/1` function to signal sumo_rest that it didn't need to check for duplicity. Maybe the easiest solution is to rename `id` as `token` everywhere in this module.